systest/build.rs

```diff
@@ -9,7 +9,7 @@ fn main() {
         .header("libssh2_sftp.h")
         .include(env::var("DEP_SSH2_INCLUDE").unwrap())
         .type_name(|s, is_struct| {
-            if (is_struct || s == "stat") && !s.starts_with("LIB") {
+            if (is_struct || s == "stat") && !s.starts_with("LIB") && s != "libssh2_struct_stat" {
```

Comment:
@alexcrichton you mean this bit?  This is just to ensure that we emit `libssh2_struct_stat` rather than `struct libssh2_struct_stat` which won't compile (it's a typedef).  It does emit code for systest and it does pass.

(However, nightly rust is broken for us due to https://github.com/rust-lang/rust/issues/63115 so it fails on appveyor currently)

Reply:
Er yes I do. I don't think I'm really understanding anything here, so seems fine to just merge when green.

Reply:
I'll add some comments here to clarify before I push this to clarify what is going on :)